### PR TITLE
Updated walkthrough with new linux install instructions

### PR DESCRIPTION
**Pull request by ekingery**
Corresponding to [PR #83](https://github.com/exercism/interactive-cli-walkthrough/pull/83).

### CODE REVIEW

File: walkthrough/index.html [hunk 1]
```diff
@@ -80,15 +82,16 @@
...
 ---
 
 ## Do you know your system’s process architecture?
 
 - [[Yes-&gt;Linux tarball]]
-- [[No-&gt;Talk to a Volunteer]]</tw-passagedata><tw-passagedata pid="7" name="Linux directory" tags="" position=""># Moving the Executable to `~/bin`
+- [[No, I would like to try the snap package-&gt;Linux - Snap]]
+- [[No, I would like talk to a volunteer-&gt;Talk to a Volunteer]]</tw-passagedata><tw-passagedata pid="7" name="Linux directory" tags="" position=""># Moving the Executable to `~/bin`
 
 Once you downloaded and extracted the
```
Comment (kotp) [inline walkthrough/index.html]: `Once you have downloaded` or `Once you download and extract`

File: walkthrough/index.html [hunk 2]
```diff
@@ -129,17 +132,14 @@
...
 
-## Do you know your process architecture (x86 (32-bit) vs x86_64 (64-bit) vs ARM)?
+## I would like to install the CLI
 
-- [[Yes-&gt;Linux tarball]]
-- [[No-&gt;Linux architecture]]</tw-passagedata><tw-passagedata pid="9" name="Linux path" tags="" position=""># Adding `~/bin` to `$PATH` in Bash
+- [[Using snap-&gt;Linux - Snap]]
+- [[Using the source code-&gt;Linux tarball]]</tw-passagedata><tw-passagedata pid="9" name="Linux path" tags="" position=""># Adding `~/bin` to `$PATH` in Bash
 
 Note: If you’re not running Bash try to adjust the
```
Comment (kotp) [inline walkthrough/index.html]: Change `you're` to `you are`

File: walkthrough/index.html [hunk 3]
```diff
@@ -331,7 +355,7 @@
 ## Do you have Homebrew installed?
 
 - [[Yes-&gt;Installing Exercism via Homebrew]]
-- [[No-&gt;Installing Homebrew]]</tw-passagedata><tw-passagedata pid="16" name="Installing Homebrew" tags="" position=""># Installing Homebrew
+- [[No-&gt;Installing Homebrew]]</tw-passagedata><tw-passagedata pid="17" name="Installing Homebrew" tags="" position=""># Installing Homebrew
 
 Now that we've found out that homebrew isn't installed on your computer yet, let's install it! If you'd like to read more on Homebrew before installing it, the &lt;a href="https://brew.sh" target="_blank"&gt;Homebrew webpage&lt;/a&gt; is a good resource.
```
Comment (kotp) [inline walkthrough/index.html]:
```suggestion
Let's install it! If you'd like to read more on Homebrew before installing it, the &lt;a href="https://brew.sh" target="_blank"&gt;Homebrew webpage&lt;/a&gt; is a good resource.
```

File: walkthrough/index.html (hunk 1 above)
Comment (ekingery) [reply walkthrough/index.html]: This is fixed now

File: walkthrough/index.html (hunk 3 above)
Comment (ekingery) [reply walkthrough/index.html]: This is fixed in the new commit.

File: walkthrough/index.html (hunk 2 above)
Comment (ekingery) [reply walkthrough/index.html]: I fixed all of the contractions in the file.

**Comment by ekingery** [comment]
> Perhaps unrelated suggestions to the changes we are making, but I try to avoid contractions in writing where I am aware that it is going to be fairly high international (multi-lingual) audience.

Sure, I try to avoid them altogether ;-). Ideally, this review would happen in the `interactive-cli-walkthrough` repo, so the changes don't have to be committed twice. As it is, I also made this PR which corresponds to the suggested changes: https://github.com/exercism/interactive-cli-walkthrough/pull/85

**Comment by rpottsoh** [comment]
This PR probably should just be closed.  Once the walk-through is recompiled it appears as though exercism/interactive-cli-walkthrough#85 fixes everything.

**Comment by ekingery** [comment]
> This PR probably should just be closed. Once the walk-through is recompiled it appears as though [exercism/interactive-cli-walkthrough#85](https://github.com/exercism/interactive-cli-walkthrough/pull/85) fixes everything.

In that case, I am misunderstanding the process. I submitted to `interactive-cli-walkthrough` because that seems like the "canonical" location of the walkthrough content. And then once that is built, you have to commit the generated .html in this repo, so it actually gets deployed to https://exercism.io/cli-walkthrough

Can you help me understand what I am missing?

**Comment by rpottsoh** [comment]
That is correct.  The person that compiles your changes (once merged) to the other repo will then deploy the new index.html file to this repo, which will render this PR moot.

**Comment by ekingery** [comment]
> That is correct. The person that compiles your changes (once merged) to the other repo will then deploy the new index.html file to this repo, which will render this PR moot.

I see. I think that may not have happened for this PR? https://github.com/exercism/interactive-cli-walkthrough/pull/83

That was the original and the bulk of the changes. I just added some additional changes here as requested above. Then I went back and also made those in the walkthrough repo, to match what was requested here. At any rate, if you can help get this pushed through to production, I would really appreciate it!

**Comment by SleeplessByte** [comment]
@rpottsoh then let's keep this open until that happens so we can track if for whatever reason it _doesn't_? (cc @kotp).

**Comment by rpottsoh** [comment]
I added a comment to exercism/interactive-cli-walkthrough#83 last night, asking that a build and update of index.html be performed.  The README of that repo instructs anyone on how to do it.

**Comment by kotp** [comment]
The merge has happened, so this can be closed.  @ekingery if you notice that the build did not happen as expected, feel free to reopen.

**Pull request closed without merging**